web/src/components/studies/Studies.tsx

```diff
@@ -36,7 +36,9 @@ export default function Studies(props: Props) {
         title: data.title,
         description: data.description ? data.description : undefined,
         data_controller_organisation: data.dataControllerOrganisation.toLowerCase(),
-        additional_study_admin_usernames: data.additionalStudyAdminUsernames.map((admin) => admin.value),
+        additional_study_admin_usernames: data.additionalStudyAdminUsernames
+          .map((admin) => admin.value.trim())
+          .map((username) => `${username}@ucl.ac.uk`),
```

Comment:
So we want to append `@ucl.ac.uk` for the prod version. But during development this suffix is actually `@arctretest.onmicrosoft.com`.

Whats the best way to approach this? Set as an env variable to say what the domain is during dev/prod?